app/services/create_batch.rb

```diff
@@ -0,0 +1,12 @@
+class CreateBatch
+  def call
+    appointment_summaries = AppointmentSummary
+                            .joins('LEFT OUTER JOIN appointment_summaries_batches ' \
```

Comment:
Can we not achieve this with Active Record without having to resort to half writing SQL?